### DISCUSSION TITLE
Large dataset on scatter plots

### DISCUSSION THREAD

**Discussion by scottquested** [recharts/recharts#3181] [Q&A] [answered]
I'm using the scatter plot chart and passing in a dataset that has around 50,000 objects in an array.

It crashes the browser if I go above 10,000. Is this a known limitation of the library?

I have tested the same data with Plotly and it renders just fine.

**Comment by ckifer** [comment, marked as answer]
Hi @scottquested 

Yeah there is an issue label with issues related to large datasets https://github.com/recharts/recharts/issues?q=is%3Aopen+is%3Aissue+label%3A%22large+dataset%22

This one has a few suggestions on how to get around this yourself https://github.com/recharts/recharts/issues/1356

Can't give you much more than that as of now - maybe eventually we'll add a mode similar to highcharts that gives a bit of a performance gain for some tradeoffs in visual.
